tools/pg_inspector/active_connections_to_human.rb

```diff
@@ -206,12 +233,40 @@ def process_miq_server(server)
       end
     end
 
+    def rails_sequence_factor
+      DEFAULT_RAILS_SEQUENCE_FACTOR
+    end
+
+    def id_to_region(id)
+      id.to_i / rails_sequence_factor
+    end
+
+    def split_id(id)
+      id = uncompress_id(id)
+
+      region_number = id_to_region(id)
+      short_id      = region_number.zero? ? id : id % (region_number * rails_sequence_factor)
+
+      return region_number, short_id
```

Comment:
We should have tests written for these methods at some point.

Reply:
Yes, I could borrow them from id_regions.